### ISSUE TITLE
Failure in submodule due to filepath mixup

### ISSUE THREAD

**Issue by peter-meinhardt** [azz/pretty-quick#90]
I am using prettier and pretty-quick in both the main repository and the submodule. Both have their own package.json with a pre-commit hook defined. When using pretty-quick inside the submodule, I get an error. The fileservice tries to acces a directory where the root path of the submodule seems to be replaced by the root path of the main repository. For example:

Main repository is at `/Users/peter.meinhardt/GIT/Customer/Project/`, submodule is at `/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module`. I try to commit a file inside the submodule at  `/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/src/lib/elements/specific-element` and will get the following error:
```husky > pre-commit (node v10.13.0)
🔍  Finding changed files since git revision 7c9ea477.
🎯  Found 1 changed file.
fs.js:115
    throw err;
    ^

Error: ENOENT: no such file or directory, scandir '/Users/peter.meinhardt/GIT/Customer/Project/src/lib/elements/specific-element'
    at Object.readdirSync (fs.js:783:3)
    at traverseFolder (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:40340:18)
    at findRoot (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:40356:10)
    at maybeParse (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:40368:16)
    at editorconfigSyncNoCache (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:40381:35)
    at editorconfigSyncNoCache (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:36212:20)
    at /Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:42627:14
    at Array.map (<anonymous>)
    at _resolveConfig (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:42626:40)
    at Function.resolveConfig.sync (/Users/peter.meinhardt/GIT/Customer/Project/projects/core-module/node_modules/prettier/index.js:42662:12)
husky > pre-commit hook failed (add --no-verify to bypass)```

**Comment by cjanietz** [comment]
I have the exact same issue and it stems from https://github.com/azz/pretty-quick/blob/master/src/scms/git.js#L8
Even though the submodule is the reference directory findUp checks in the directory above. I will submit a PR

**Comment by azz** [comment]
I'm not really understanding the fix, @cjanietz. I thought `find-up` would look in the submodule directory before searching up the FS tree?

**Comment by cjanietz** [comment]
.git is not a directory but a file in a submodule, therefore its either checking locally or making findUp search for file and dir @azz

**Comment by cjanietz** [comment]
@azz Can we please get this sorted out? Anything else you would like different?

**Comment by azz** [comment]
Sorry @cjanietz, missed the notification. Fixed in #90.
